pages/linux/trayer.md

```diff
@@ -0,0 +1,28 @@
+# trayer
+
+> A lightweight GTK-2 based systray for UNIX desktop.
+> More information: <https://github.com/sargon/trayer-srg>.
+
+- Run trayer:
+
+`trayer`
+
+- Position trayer to the top:
+
+`trayer --edge {{top}}`
+
+- Provide height as 32px and width as 10% of screen resolution:
+
+`trayer --width {{10}} height {{ 32 }}`
```

Comment:
```suggestion
`trayer --width {{10}} --height {{32}}`
```
Shouldn't the height be a flag too here?

Reply:
yes... I forgot. Thanks for that